core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -636,11 +646,13 @@ object TopicCommand extends Logging {
       if(options.has(createOpt))
           CommandLineUtils.checkInvalidArgs(parser, options, replicaAssignmentOpt, Set(partitionsOpt, replicationFactorOpt))
       CommandLineUtils.checkInvalidArgs(parser, options, reportUnderReplicatedPartitionsOpt,
-        allTopicLevelOpts -- Set(describeOpt) + reportUnderMinIsrPartitionsOpt + reportUnavailablePartitionsOpt + topicsWithOverridesOpt)
+        allTopicLevelOpts -- Set(describeOpt) + reportUnderMinIsrPartitionsOpt + reportUnavailablePartitionsOpt + reportAtMinIsrPartitionsOpt + topicsWithOverridesOpt)
       CommandLineUtils.checkInvalidArgs(parser, options, reportUnderMinIsrPartitionsOpt,
-        allTopicLevelOpts -- Set(describeOpt) + reportUnderReplicatedPartitionsOpt + reportUnavailablePartitionsOpt + topicsWithOverridesOpt + zkConnectOpt)
+        allTopicLevelOpts -- Set(describeOpt) + reportUnderReplicatedPartitionsOpt + reportUnavailablePartitionsOpt + reportAtMinIsrPartitionsOpt + topicsWithOverridesOpt + zkConnectOpt)
```

Comment:
Nit: 
We repeatedly add `reportUnderReplicatedPartitionsOpt`, `reportUnavailablePartitionsOpt`, `reportAtMinIsrPartitionsOpt` and `reportUnderMinIsrPartitionsOpt` to the set of disallowed options, and it is messy to read (and a  . Maybe create a new set like `allReplicationReportOpts` and use this everywhere?

Reply:
Great idea. Thanks!